pandas/tests/io/parser/test_compression.py

```diff
@@ -0,0 +1,212 @@
+# -*- coding: utf-8 -*-
+
+"""
+Tests compressed data parsing functionality for all
+of the parsers defined in parsers.py
+"""
+
+import bz2
+import gzip
+import os
+import zipfile
+
+import pytest
+
+import pandas.compat as compat
+
+import pandas as pd
+import pandas.util.testing as tm
+
+
+def lzma_file():
+    """
+    Try to load the `LZMAFile` class from `backports.lzma`.
+
+    Returns
+    -------
+    klass : type or None
+    """
+    try:
+        lzma = compat.import_lzma()
+    except ImportError:
+        lzma = None
+
+    return getattr(lzma, "LZMAFile", None)
```

Comment:
maybe these can put in a more generic place (e.g. useable by other io tests); future PR though

Reply:
Agreed.  I didn't see any other uses of `lzma` in these tests at the moment.